cpp/include/cudf/lists/list_view.cuh

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cuda_runtime.h>
+#include <cudf/types.hpp>
+
+/**
+ * @file list_view.cuh
+ * @brief Class definition for cudf::list_view.
+ */
+
+namespace cudf {
+
+class list_view {
```

Comment:
Since this is only used as a tag type for now, I'd get rid of any of the contents and make it an empty tag type
```suggestion
class list_view {};
```